lib/ex_doc/formatter/html/autolink.ex

```diff
@@ -447,6 +447,37 @@ defmodule ExDoc.Formatter.HTML.Autolink do
     end
   end
 
+  defp replace_fun(:mix_task, :elixir, _link_type, options) do
```

Comment:
`since we are only supporting :normal links, this line can read:

`  defp replace_fun(:mix_task, :elixir, :normal, options) do`